src/yesql/generate.clj

```diff
@@ -72,21 +72,27 @@
 ;; case, we only ever use one group, so we'll unpack the
 ;; single-element list with `first`.
 (defn execute-handler
-  [db sql-and-params call-options]
-  (first (jdbc/execute! db sql-and-params)))
+  [db sql params call-options]
+  (first (jdbc/execute! db (rewrite-query-for-jdbc sql params))))
 
 (defn insert-handler
-  [db [statement & params] call-options]
-  (jdbc/db-do-prepared-return-keys db statement params))
+  [db sql params call-options]
+  (if (vector? params)
+    (let [table-string-position 0
```

Comment:
so these are magic numbers in the real sense - it works for 90% of the use cases, but it wont work if CTEs or MySQL priorities are used. Thinking about the best way to parse for table names in a sql statement.